app/actions/index.js

```diff
@@ -23,6 +25,10 @@ window.fetch = new Fetch({
   binaryContentTypes: ["image/", "video/", "audio/", "application/gzip"]
 }).build();
 
+const obsdb = createOsmp2p("obs");
+const osmdb = createOsmp2p("osm");
+const osm = osmp2p(obsdb, osmdb);
```

Comment:
there won't be lock file errors as asyncstorage has different behavior, but i'm removing that db instance from Observations/map.js